mobile/src/main/java/org/openhab/habdroid/ui/IntroActivity.kt

```diff
@@ -93,17 +95,23 @@ class IntroActivity : AppIntro() {
      * @param imageDrawable
      */
     private fun addSlide(@StringRes title: Int, @StringRes description: Int, @DrawableRes imageDrawable: Int) {
-        @ColorInt val greyColor = ContextCompat.getColor(this, R.color.grey_300)
-        @ColorInt val blackColor = ContextCompat.getColor(this, R.color.black)
+        val colorText = resolveThemeColor(R.attr.colorOnBackground)
+        val colorBackground = resolveThemeColor(android.R.attr.windowBackground)
 
         addSlide(AppIntroFragment.newInstance(getString(title),
             null, // Title font: null => default
             getString(description),
             null, // Description font: null => default
             imageDrawable,
-            greyColor, // Background color
-            blackColor, // Title color
-            ContextCompat.getColor(this, R.color.black))) // Description color
+            colorBackground, // Background color
+            colorText, // Title color
+            colorText)) // Description color
+    }
+
+    private fun resolveThemeColor(@AttrRes colorAttr: Int): Int {
```

Comment:
I'd make a extension function of `context` out of this one and use it in other places as well, but I can do that in a separate PR.

Reply:
I thought about this (and actually started), but wasn't sure what to make out of this check: https://github.com/openhab/openhab-android/blob/d4b6d6dc44ac54631f9ffc5c88ac91877b65ac9a/mobile/src/main/java/org/openhab/habdroid/ui/AbstractBaseActivity.kt#L85 ... do you remember what the point of that check is?

Reply:
The background might be a drawable. This check makes sure we get a color. Copied from https://stackoverflow.com/a/14468034/13139982

Reply:
Hmm, OK. I wonder what the best way to deal with that difference is, then:
- Make the method signature e.g. `resolveThemedColor(attr: Int): Int?` and return null if it's not a color or
- Have two different methods (`resolveThemedColor` and `resolveThemedColorWithFallback`), where the latter allows passing a fallback color

The former might be a bit more cumbersome to use due to needed null checks ... or pass a fallback color instead of returning a nullable value and pass 0 as fallback if we're sure the color is resolvable?

Reply:
Done.